views/check-your-answers.html

```diff
@@ -28,6 +28,57 @@ <h1 class="govuk-heading-xl">
         Check your objection application
       </h1>
 
+      <h2 class="govuk-heading-m">
+        Your details
+      </h2>
+
+      {{ govukSummaryList({
+        rows: [
+          {
+            key: {
+              text: "Email address"
+            },
+            value: {
+              text: objection.created_by.email
+            }
+          },
+          {
+            key: {
+              text: "Full name"
+            },
+            value: {
+              text: objection.created_by.fullName
+            },
+            actions: {
+              items: [
+                {
+                  href: "/objecting-entity-name/?id=1",
```

Comment:
Looks like this is adding the 'change' link but not in screenshot on story or part of description.